Sources/SQL/Core/main.swift

```diff
@@ -0,0 +1,141 @@
+extension Table {
+    public static func f(field: Field) -> DeclaredField {
+        return self.field(field)
+    }
+}
+
+
+struct Artist {
+    let id: Int?
+    // Note the optionality. Fields that are marked `NOT NULL` in your schema should be non-optional.
+    var name: String
+    var genre: String?
+
+    // You probably want your own initializer
+    init(name: String, genre: String) {
+        self.id = nil
+        self.name = name
+        self.genre = genre
+    }
+}
+
+
+extension Artist: Table {
```

Comment:
Bottomline here is that I want to have my models be pure `struct`s. And anything else I do with them related to persistence should have persistence parlance.